core/sqf/sql/scripts/lobstart

```diff
@@ -30,8 +30,8 @@ let totalNodes=`grep 'Up' $tmp_node_status | wc -l`
 while [ ${count} -lt $totalNodes ]
 do
 sqshell -a > /dev/null << eof
-set {process \zlobsrv${count} } PERSIST_RETRIES=10,60
-set {process \zlobsrv${count} } PERSIST_ZONES=0,1
+set {process \$zlobsrv${count} } PERSIST_RETRIES=10,60
+set {process \$zlobsrv${count} } PERSIST_ZONES=${count}
 exec {name \$zlobsrv${count}, nowait, nid ${count}, out stdout_\$ZLOBSRV${count} } mxlobsrvr
```

Comment:
Yes the lob processes will come up when a down node is reintegrated thanks to the change made in gensq.pl. Stylistically it could be better to have the lobstart and the lob config db scripts together. But functionally this setup works just fine. So  would prefer not to make further changes to rework this at this point.